efficientdet/dataloader.py

```diff
@@ -406,10 +409,16 @@ def _prefetch_dataset(filename):
       dataset = dataset.shuffle(64)
```

Comment:
tf.dataset.list_files already shuffles the dataset

Reply:
That's to shuffle the files, this is to shuffle the examples?

Reply:
oh it is only usefull if we have one single record file,else i dont see any use of it. because batch_size is one here.